src/main/ui/app/route/About.jsx

```diff
@@ -6,18 +6,33 @@ export default class About extends Component {
     render() {
         return (
                 <div>
-                    <h3>About</h3>
+                    
+                    <h1>About</h1>
 
-                    <p>
-                        <Link to="/indexingStatus">Indexing Status</Link>
+                    <h3>1)How to add your module?</h3>
+
+                    <p> Adding your module to Add Ons for indexing is as easy as creating a pull request!
+                        Before we do that, we need to ensure that your module has been hosted on one of the sites that Add Ons currently supports indexing from: <br/>
+                        a) Bintray <br/>
+                        b) Modulus <br/>
+                    <br/>
+                        Currently we only support these 2 servers but in the future we sure will add more based on user demand! Currently, we are working on adding support for Github Releases as well!
```

Comment:
This line is too much information. :-)

Instead, in the contributing section below, let's add a link like "Discuss how this tool works and/or propose new features at (link)". And that should link to the new Talk category that I'm suggesting at https://talk.openmrs.org/t/create-a-category-for-the-add-on-index-project/11700